src/main/java/org/spongepowered/api/world/generation/feature/Feature.java

```diff
@@ -55,22 +55,35 @@ public interface Feature extends DefaultedRegistryValue {
     DataView toContainer();
 
     /**
-     * Places the feature at given position and world
-     *
-     * @param world The world
-     * @param pos The position
-     *
-     * @return true when the feature was successfully placed
-     */
-    boolean place(ServerWorld world, Vector3i pos);
```

Comment:
I don't see a reason why this couldn't remain, if a callback can be optionally an overloaded method.